zanata-model/src/main/java/org/zanata/model/HLocaleMember.java

```diff
@@ -63,6 +65,11 @@ public HLocaleMember(HPerson person, HLocale supportedLanguage,
         setTranslator(isTranslator);
         setReviewer(isReviewer);
         setCoordinator(isCoordinator);
+        userFriendlyToString =
+                String.format(
+                        "Locale membership(person=%s, isTranslator=%s, isReviewer=%s, isCoordinator=%s)",
+                        person.getName(), isTranslator, isReviewer,
```

Comment:
This is user display name. Username might be more useful to show instead or as well.